src/main/java/de/komoot/photon/OsmTag.java

```diff
@@ -0,0 +1,22 @@
+package de.komoot.photon;
+
+/**
+ * Created by Sachin Dole on 2/6/2015.
+ */
+public class OsmTag {
```

Comment:
if you are lazy you can annotate the class with `@Data` from lombok  that writes setter and getter as well as the constructor automatically.